src/common/modal/index.jsx

```diff
@@ -28,7 +27,7 @@ const Modal =({ title, show, onClose, onSubmit, children, cname })=> {
           { children }
         </div>
         <div className={`modal-${cname}-footer`}>
-          <button className="btn-default-light btn-size--sm" onClick={ onClose }><GoX size="16px" className="icon" /> Cancel</button>
+          <button className="btn-default-light btn-size--sm" onClick={ onClose }><GoX size="16px" className="icon" /> Close</button>
```

Comment:
This is a common modal @mayukh551, so do we want this change?
or can we make it a prop based?

Reply:
I think the change is not necessary.
Will change it from "Close" to "Cancel"